lib/fog/core/model.rb

```diff
@@ -21,16 +21,7 @@ def initialize(new_attributes = {})
     end
 
     def inspect
-      Thread.current[:formatador] ||= Formatador.new
-      data = "#{Thread.current[:formatador].indentation}<#{self.class.name}"
-      Thread.current[:formatador].indent do
-        unless self.class.attributes.empty?
-          data << "\n#{Thread.current[:formatador].indentation}"
-          data << self.class.attributes.map { |attribute| "#{attribute}=#{send(attribute).inspect}" }.join(",\n#{Thread.current[:formatador].indentation}")
-        end
-      end
-      data << "\n#{Thread.current[:formatador].indentation}>"
-      data
+      Fog::Formatador.new(self)
```

Comment:
@geemus `display` kind of implies it is gonna just print something, but the class is gonna do more then just print. What about `pretty_print` or something like that?